### DISCUSSION TITLE
To module authors, please use more descriptive names for your modules (note to 2sxc)

### DISCUSSION THREAD

**Discussion by Tony20221** [oqtane/oqtane.framework#2762] [Ideas]
I installed 2sxc and didn't find the name of the module in the list of modules in the page admin. Then I figured out that it created two modules, App and Content. I only made the connection from the version number.

To module authors, please use more descriptive names in your names. For 2csx, something like '2sxc App ' and '2sxc Content' would 
 have helped me a lot.

**Comment by iJungleboy** [comment]
@Tony20221 sorry for the confusion.

A reason is that in the past (on DNN) we've had editors who were confused by the name `2sxc` so we try to keep it "under the covers" 😉

But I understand that this stumped you...
